### PR TITLE
Awesome books: With ES6

### PR DESCRIPTION
**Pull request by ha-manel**
## 🎯 In this branch I:
- Created `modules` folder which contains the `books` class and `luxon` script.
- Added date and time using `luxon library`.
- Refactored my methods and functions to arrow functions.
- Used ES6 `let` and `const`.

### CODE REVIEW

File: modules/books.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    const book = {
+      title,
+      author,
+    };
+    this.booksArray.push(book);
+    localStorage.setItem('books', JSON.stringify(this.booksArray));
+    this.displayBooks();
+  }
+
+  removeBook = (index) => {
```
Comment (Meltrust) [inline modules/books.js:16]: - [ ] After many tries, it's impossible for me to remove the very first book on the list.  The remove button is only working for me in all the other books but not on the first one.  Tested both on Firefox and Chrome.

Please, kindly correct this bug so the app works as intended. 💪

**Review by Meltrust — CHANGES_REQUESTED** [review]
Hi @ha-manel,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



### To highlight:

- Nice styling✔️ 

You can do it

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: modules/books.js (hunk 1 above)
Comment (ha-manel) [reply modules/books.js:16]: @Meltrust thank you for pointing out the bug, turned out that when you create the event listeners while creating the HTML elements all inside a loop, the previous event listeners are lost. I fixed it 😄

**Review by dasileker — APPROVED** [review]
### **Hi @ha-manel,** 

### **Congratulations! 🎉**

Your project is complete!  There is nothing else to say other than... it's time to merge it 🚀 : 
 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**